Automattic/wp-calypso
# A/B Test: Auto-add privacy protection and no-popup

Deploy winning variation.

Before:
![privacy old](https://user-images.githubusercontent.com/1103398/29637885-d6fb8ad6-8855-11e7-8b2d-d26fc131216f.png)
After:
![image](https://user-images.githubusercontent.com/12596797/29717134-0fedeb3c-897c-11e7-9338-394291d42832.png)

Test:
- Domains -> Add
- Select a domain
- Make sure it's auto-added
- Make sure the privacy form on details form works

Test Signup:
- http://calypso.localhost:3000/start/
- Get to the domains screen
- Pick a domain with plan
- Make sure privacy is added
-----------------------------------------
- http://calypso.localhost:3000/start/
- Free plan
- Get to the domains screen
- Pick the default wp.com subdomain
- Make sure you end on checkout properly
-----------------------------------
- Go to the site you created ^
- Plan 
- Pick a plan
- Make sure you end on checkout properly

Test /domains:
- calypso.localhost:3000/start/domain-first?new=testing-privacy-auto-add-hehe-666.blog
- Just buy a domain
- make sure privacy is auto added

-------------------------------------

Test PR: https://github.com/Automattic/wp-calypso/pull/17559
